.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '30 3 * * 2'
+
+name: Publish
+
+jobs:
+
+  docker-publish:
+    name: Publish Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build Docker image
+        run: |
+          docker build \
+            --no-cache \
+            -t ghcr.io/spaceapi-community/spaceapi-sensor-logger:latest \
+            -t ghcr.io/spaceapi-community/spaceapi-sensor-logger:v1 \
```

Comment:
We hard-code the tag in the CI file? It was like this before, so nothing to be changed in this PR.

Reply:
Apparently? I've never touched this project before, I'm just migrating the CI pipeline and images.